pkg/network/ovn_kubernetes.go

```diff
@@ -1544,6 +1549,16 @@ func bootstrapOVN(conf *operv1.Network, kubeClient cnoclient.Client, infraStatus
 		ipsecStatus.Version = ipsecDaemonSet.GetAnnotations()["release.openshift.io/version"]
 	}
 
+	// If we are upgrading from 4.13 -> 4.14 set new API for IP Forwarding mode to Global.
+	// This is to ensure backwards compatibility.
+	if masterStatus != nil {
```

Comment:
is this like the IC hack? can be removed in 4.15?

Reply:
yeah, once we past 4.14 the api will exist, so we will remove this. In that case a user will be upgrading from 4.14 -> 4.15 and already have their API set.